src/core/tests/CMakeLists.txt

```diff
@@ -75,10 +75,6 @@ set(SRC
     partial_shape.cpp
     pass_config.cpp
     pass_manager.cpp
-    pass/serialization/cleanup.cpp
```

Comment:
Serialize transformation is a part of public API and it is a part of core component. Why did you move these tests to inference_engine func tests? Please revert these changes.

Reply:
done